### ISSUE TITLE
One maintainer of ApiGen/ApiGen here: Is this still a thing?

### ISSUE THREAD

**Issue by jankal** [apigen-ci/apigen-ci#1]
@mmeyer2k If this has a clear aim to become a docs generation CI, we might add it to apigen org to support and develop besides PHP 7.1 support as a new feature.

@TomasVotruba What do you think?

**Comment by mmeyer2k** [comment]
Yes. If anyone wants to keep this alive I would support that. I had this running but then I had issues with my hosting and I just let it die. Have been too busy with work to give it any attention. But if there is real interest in it I would try to find some time to get it running again.

**Comment by jankal** [comment]
I was just curious cause I also had this type of idea some time ago.

Maybe this would be a new entry point for probably ApiGen 5 usage.

**Comment by mmeyer2k** [comment]
Yeah it was a pretty useful tool, I just never really tried to get the word out about it. It was a side project and not very polished. 

I'm open for anything. I can see the benefit of putting this project under the umbrella of apigen.

I also have the domain apigen.ci =D

**Comment by jankal** [comment]
Yep. Very nice. I will deploy this for myself to evaluate tomorrow.

I may suggest an update to Laravel 5.4.

Therefor: It would be nice to have the `composer.json` updated so Laravel components will be installed.

If @TomasVotruba and @kukulich agree on moving this to the organization it could be moved and would be a good additional feature of apigen.

**Comment by mmeyer2k** [comment]
I coded it as a laravel package so I could deploy it easily in my infrastructure. But, yes, if it becomes a standalone package it should be restructured and updated to the latest version.

I can get you the DB schema too. Let me know if you need any help!
